ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_random.py

```diff
@@ -162,3 +162,31 @@ def test_paddle_uniform_(
         max=max,
         seed=seed,
     )
+
+
+@handle_frontend_test(
+    fn_tree="paddle.standard_normal",
+    input_dtypes=st.sampled_from([["int32"], ["int64"]]),
```

Comment:
What does this do? Does `get_shape` ever return something other than `int`s that `input_dtypes` have to be included?

Reply:
`shape` kwarg always contain `int`, so I set `input_dtypes` to be one of `int32` or `int64` since `input_dtypes` is not an optional kwarg for `test_frontend_function`.

Reply:
Ah got it, makes sense :+1: